### DISCUSSION TITLE
FIBO schema.org properties and FIBO OWL alignment proposal

### DISCUSSION THREAD

**Discussion by sdml** [edmcouncil/fibo#1392] [General]
- [Thing](http://pending.schema.org/Thing)
   - [Property](http://pending.schema.org/Property)
      - [annualPercentageRate](http://pending.schema.org/annualPercentageRate)
      - [feesAndCommissionsSpecification](http://pending.schema.org/feesAndCommissionsSpecification)
      - [interestRate](http://pending.schema.org/interestRate) -> in FIBO - [hasInterestRate](https://spec.edmcouncil.org/fibo/ontology/FBC/DebtAndEquities/Debt/hasInterestRate)
      - [identifier](http://pending.schema.org/identifier)
         - [leiCode](http://pending.schema.org/leiCode) -> in FIBO - [hasTag](https://spec.edmcouncil.org/fibo/ontology/FND/Relations/Relations/hasTag)
      - [duration](http://pending.schema.org/duration)
         - [loanTerm](http://pending.schema.org/loanTerm) -> in FIBO - [hasTerm](https://spec.edmcouncil.org/fibo/ontology/FBC/FinancialInstruments/FinancialInstruments/hasTerm)
      - [requiredCollateral](http://pending.schema.org/requiredCollateral) -> in FIBO - [isCollateralizedBy](https://spec.edmcouncil.org/fibo/ontology/FBC/DebtAndEquities/Debt/isCollateralizedBy)
      - [accountMinimumInflow](http://pending.schema.org/accountMinimumInflow)
      - [accountOverdraftLimit](http://pending.schema.org/accountOverdraftLimit)
      - [amount](http://pending.schema.org/amount) -> in FIBO - [hasMonetaryAmount](https://spec.edmcouncil.org/fibo/ontology/FND/Accounting/CurrencyAmount/hasMonetaryAmount)
      - [bankAccountType](http://pending.schema.org/bankAccountType)
      - [beneficiaryBank](http://pending.schema.org/beneficiaryBank)
      - [cashBack](http://pending.schema.org/cashBack)
      - [contactlessPayment](http://pending.schema.org/contactlessPayment)
      - [currency](http://pending.schema.org/currency) -> in FIBO - [hasCurrency](https://spec.edmcouncil.org/fibo/ontology/FND/Accounting/CurrencyAmount/hasCurrency)
      - [currentExchangeRate](http://pending.schema.org/currentExchangeRate)
      - [domiciledMortgage](http://pending.schema.org/domiciledMortgage) -> in FIBO - [isDomiciledIn](https://spec.edmcouncil.org/fibo/ontology/FND/Organizations/FormalOrganizations/isDomiciledIn)
      - [downPayment](http://pending.schema.org/downPayment)
      - [earlyPrepaymentPenalty](http://pending.schema.org/earlyPrepaymentPenalty)
      - [exchangeRate](http://pending.schema.org/exchangeRate)
      - [exchangeRateSpread](http://pending.schema.org/exchangeRateSpread)
      - [floorLimit](http://pending.schema.org/floorLimit) -> in FIBO - [hasCreditLimit](https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanContracts/LoanCore/hasCreditLimit) (this is incorrect as pointed out by @VladimirAlexiev below)
      - [gracePeriod](http://pending.schema.org/gracePeriod) -> in FIBO - [hasGracePeriod](https://spec.edmcouncil.org/fibo/ontology/FBC/DebtAndEquities/CreditEvents/hasGracePeriod)
      - [loanMortgageMandateAmount](http://pending.schema.org/loanMortgageMandateAmount) - [loanMortgageMandateAmount](https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/MortgageLoans/loanMortgageMandateAmount)
      - [loanPaymentAmount](http://pending.schema.org/loanPaymentAmount) -> in FIBO - [paymentAmount](https://spec.edmcouncil.org/fibo/ontology/LOAN/LoansTemporal/LoansStatus/paymentAmount)
      - [loanPaymentFrequency](http://pending.schema.org/loanPaymentFrequency)
      - [loanRepaymentForm](http://pending.schema.org/loanRepaymentForm) -> in FIBO - [hasRepaymentTerms](https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/DebtInstruments/hasRepaymentTerms)
      - [loanType](http://pending.schema.org/loanType)
      - [monthlyMinimumRepaymentAmount](http://pending.schema.org/monthlyMinimumRepaymentAmount)
      - [numberOfLoanPayments](http://pending.schema.org/numberOfLoanPayments)
      - [recourseLoan](http://pending.schema.org/recourseLoan) -> in FIBO - [recourseLoan](https://spec.edmcouncil.org/fibo/ontology/LOAN/Loans/LoanBasicTerms/recourseLoan)
      - [renegotiableLoan](http://pending.schema.org/renegotiableLoan) -> in FIBO - [isNegotiable](https://spec.edmcouncil.org/fibo/ontology/FBC/FinancialInstruments/FinancialInstruments/isNegotiable)

**Comment by sdml** [comment]
Is this mapping OK? And how to fill in missing terms?

**Comment by VladimirAlexiev** [comment]
It seems that `floorLimit` "amount of money above which credit card transactions must be authorized" is the "opposite" of `hasCreditLimit`
